andela/travel_tool_front
# #160233501 Assigning manager role to user

#### What does this PR do?
Replaces the manager field in the new request form with a dropdown and populates it with a list of managers

#### Description of Task to be completed?
- Create actions, sagas, reducers and services to fetch users with manger roles from the backend
- Test all actions reducers, sagas and services
- Change the manager field in the new request form to a dropdown
- Populate the manager dropdown with the names of managers
- Fix some errors in some components
- Fix failing tests in other components


#### How should this be manually tested?
- Pull the branch
- Change directory into the folder `cd travel_tool_front`
- Run `make start`
- Start the backend of the application
- Assign yourself the manager role in the backend- see (https://github.com/andela/travel_tool_back/pull/31)
- Open the browser and navigate to the Requests URL `http://127.0.0.1:3000/requests`
- Click on the 'New Request' button, you should see your name as one of the managers in the dropdown


#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160233501](https://www.pivotaltracker.com/story/show/160233501)


#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-09-04 at 5 44 04 pm" src="https://user-images.githubusercontent.com/25195857/45046489-35c82380-b06e-11e8-9649-67051bbf96a2.png">


#### Questions:
None
